library/cables/XLPE_1000m_220kV_dynamic.yaml

```diff
@@ -0,0 +1,10 @@
+ac_resistance: 0.16        # ohm/km
+capacitance: 190           # nF/km
+conductor_size: 1000       # mm^2
+cost_per_km: 1020000        # $ (20% adder over XLPE_1000m_220kV)
```

Comment:
Extremely minor, but there is an extra space in front of the comment relative to the others.